paddle/fluid/framework/ir/mkldnn/cpu_quantize_placement_pass.cc

```diff
@@ -32,6 +32,9 @@ void CPUQuantizePlacementPass::ApplyImpl(ir::Graph* graph) const {
                                        "fused_conv2d",
                                        "fused_conv3d",
                                        "fused_matmul",
+                                       "fused_elementwise_add",
+                                       "fused_elementwise_mul",
+                                       "fused_elementwise_sub",
```

Comment:
Please remove if-block cpu_quantize_placement_pass.cc:90
`// Remove this condition when all fused_elementwise ops are merged`
It was temporary solution, which is solved by your PR